### ISSUE TITLE
LES perturbations diagnostic variable attributes

### ISSUE THREAD

**Issue by kpamnany** [CliMA/ClimateMachine.jl#1570]
# Description

Each of the diagnostic variables named [here](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Diagnostics/atmos_les_default_perturbations.jl#L101-L123) need definitions added [here](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Diagnostics/variables.jl#L463).

# For CLIMA Developers

- [ ] This feature can be added (if it cannot be, explain why in a comment below -- lack of technical expertise, not relevant to the scope of this project, too ambitious)
- [ ] There is a timeline for when this feature can be implemented
- [ ] The feature has been (or will be) implemented (Please link the PR)

**Comment by szy21** [comment]
Do we need those perturbations in the diagnostics?

**Comment by kpamnany** [comment]
They were requested: #1243.

**Comment by szy21** [comment]
@akshaysridhar I'm a bit confused about what these perturbations are used for. The horizontal mean of the perturbations will be zero?

**Comment by kpamnany** [comment]
They are perturbations _from_ the horizontal mean.

**Comment by szy21** [comment]
> They are perturbations _from_ the horizontal mean.

Right, but do we save the horizontal mean of the perturbations in the output, or we save the perturbations at each point?

**Comment by kpamnany** [comment]
It outputs the perturbations at each point.
